workspaces/website/src/blocks/ListCardItems.tsx

```diff
@@ -52,6 +53,7 @@ Props): JSX.Element {
                 key={`${card.title}-${i}`}
                 description={card.description}
                 title={card.title}
+                discordHandle={card.discordHandle}
```

Comment:
this is not the handle but the url! the same for twitter. Maybe you can start improving some code snippets
```tsx
card_list_items?.map(({ website_url, ...card }, index) => (
  <ListCard
    href={website_url}
    key={`${card.title}-${index}`}
    {...card}
  />
))
```